apps/dotcms-ui/src/app/view/components/_common/dot-site-selector/dot-site-selector.component.spec.ts

```diff
@@ -193,7 +193,7 @@ describe('SiteSelectorComponent', () => {
         comp.handleFilterChange(filter);
 
         expect(paginatorService.getWithOffset).toHaveBeenCalledWith(0);
-        expect(paginatorService.filter).toEqual(filter);
+        expect(paginatorService.filter).toEqual(`*${filter}`);
```

Comment:
Shouldn't we add the wildcard at the end of the search term as well?

Reply:
not needed